rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -35,6 +38,8 @@ BuildRequires: ruby(release)
 %endif
 BuildRequires: rubygems-devel
 BuildArch: noarch
+Obsoletes: tfm-rubygem(smart_proxy_remote_execution_ssh_core) < 0.1.4
```

Comment:
ditto, use variable expansion here as this is wrong on Fedora, and the line below wrong on EL